packages/lesswrong/components/comments/CommentWithReplies.tsx

```diff
@@ -35,8 +36,13 @@ const CommentWithReplies = ({
   commentNodeProps,
   classes,
 }: CommentWithRepliesProps) => {
-  const [maxChildren, setMaxChildren] = useState(initialMaxChildren);
+  const { hash: focusCommentId } = useLocation();
+
+  const commentId = focusCommentId.slice(1) || null;
+  const startExpanded = !!comment.latestChildren.find(c => c._id === commentId)
```

Comment:
Could you use [`.some()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) instead?